libc/utils/docgen/docgen.py

```diff
@@ -176,10 +180,22 @@ def print_impl_status_rst(header: Header, api: Dict):
         print_functions_rst(header, api["functions"])
 
 
+# This code implicitly relies on docgen.py being in the same dir as the json
+# files and is likely to need to be fixed when re-integrating docgen into
+# hdrgen.
+def get_choices():
```

Comment:
nit:
```
def get_choices() -> List:
```

Reply:
ba1d7489eb30

good catch, I'm still not used to putting explicit types in my python...